### PR TITLE
v4.3.5 RNs

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-3972
| Versions      | all
| Edition       | all

Note: merge the same note to the next 3.3 and 4.4 releases.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/release_notes/ibexa_dxp_v4.3.md [hunk 1]
```diff
@@ -159,3 +159,13 @@ product-related responses.
...
 You can now retrieve customer group by implementing the `Ibexa\Contracts\ProductCatalog\CustomerGroupResolverInterface` interface and tagging it with `ibexa.product_catalog.customer_group.resolver`.
+
+## v4.3.5
+
+- The `UserService::updateUserPassword` method now throws `ContentFieldValidationException`
+with array format accessible via `ContentFieldValidationException::getFieldErrors`.
+The format of the returned array is:
+
+```
+array<<int fieldId>, array<<string language code>, array<ValidationError>>
```
Comment (alongosz) [inline docs/release_notes/ibexa_dxp_v4.3.md:170]: Whoops, my mistake:
```suggestion
array<<int fieldId>, array<<string language code>, array<\Ibexa\Contracts\Core\FieldType\ValidationError>>>
```

File: docs/release_notes/ibexa_dxp_v4.3.md [hunk 2]
```diff
@@ -159,3 +159,13 @@ product-related responses.
 #### Ability to retrieve a customer group
 
 You can now retrieve customer group by implementing the `Ibexa\Contracts\ProductCatalog\CustomerGroupResolverInterface` interface and tagging it with `ibexa.product_catalog.customer_group.resolver`.
+
+## v4.3.5
+
+- The `UserService::updateUserPassword` method now throws `ContentFieldValidationException`
+with array format accessible via `ContentFieldValidationException::getFieldErrors`.
+The format of the returned array is:
```
Comment (alongosz) [inline docs/release_notes/ibexa_dxp_v4.3.md:167]: I would actually rephrase it to emphasize that the format has changed. Sounds too much like completely new exception.

_Background: The method always threw that exception on error, but the array structure/shape was incorrect._

**Review by alongosz — CHANGES_REQUESTED** [review]

**Review by alongosz — APPROVED** [review]

**Review by MagdalenaZuba — APPROVED** [review]
